pkg/cloudprovider/controller/nodecontroller.go

```diff
@@ -34,6 +34,32 @@ import (
 	"github.com/golang/glog"
 )
 
+const (
+	// The constant is used if sync_nodes_status=False. NodeController will not proactively
+	// sync node status in this case, but will monitor node status updated from kubelet. If
+	// it doesn't receive update for this amount of time, it will start posting "NodeReady==
+	// ConditionNone". The amount of time when NodeController start evicting pods is controlled
```

Comment:
The amount of time when NodeController start evicting -> The amount of time before which NodeController starts evicting

Also, shouldn't this say NodeReady==ConditionUnknown, not NodeReady==ConditionNone?